features/step_definitions/icon_steps.rb

```diff
@@ -34,6 +34,12 @@
   step %{I press "Update"}
 end
 
+When /^I delete the icon from my psued$/ do
```

Comment:
For new steps, we lean towards Cucumber expressions: https://github.com/otwcode/otwarchive/wiki/Automated-Testing#our-cucumber-standards (at least where they're supported). I have a Rubocop rule in the works, but it's not merged just yet.

(Also it looks like there's a typo in pseud)

```suggestion
When "I delete the icon from my pseud" do
```

Reply:
Oh thank you, I'll remember this for future tests, and good catch on that misspelling. Changed here and in the feature file.